Uplift/Views/Cells/GymWeekCell.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  GymWeekCell.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/6/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymWeekCell: UICollectionViewCell {
+
+    // MARK: - Views
+    private var dayLabel: UILabel!
+    private var backCircle: UIView!
+
+    // MARK: - Info
+    /// Weekday this cell represents
+    var day: WeekDay?
+    /// Is this day today
```

Comment:
this comment doesn't make a ton of sense -- could we call the variable `isToday` and just get rid of the comment?

Reply:
yea I realized -- let's get that one merged so we can pull all the PR fixes into this PR once it's merged